testservices/novaservice/service.go

```diff
@@ -143,6 +150,53 @@ func New(hostURL, versionPath, tenantId, region string, identityService, fallbac
 	return novaService
 }
 
+func NewNovaOnly(cred *identity.Credentials, auth identity.AuthMode) (*Nova, []string) {
```

Comment:
nitpic - all exported methods should have a comment, here and a few other places